.bintray.sh

```diff
@@ -1,27 +1,38 @@
+BINTRAY_REPO_NAME_SUFFIX="$1"
+BINTRAY_PKG_NAME="$2"
+VERSION="$3"
+OUTPUT_TARGET="bintray_${BINTRAY_REPO_NAME_SUFFIX}.yml"
+
 DATE=`date +%Y-%m-%d`
 
+# TODO (above): re-enable PGP signing
+
 YAML="{
   \"package\": {
-    \"name\": \"ponyc\",
-    \"repo\": \"$1\", 
+    \"repo\": \"ponyc-$BINTRAY_REPO_NAME_SUFFIX\",
+    \"name\": \"$BINTRAY_PKG_NAME\",
     \"subject\": \"ponylang\"
   },
   \"version\": {
-    \"name\": \"$2\",
-    \"desc\": \"ponyc release $2\",
+    \"name\": \"$VERSION\",
+    \"desc\": \"ponyc release $VERSION\",
     \"released\": \"$DATE\",
-    \"vcs_tag\": \"$2\",
-    \"gpgSign\": true
+    \"vcs_tag\": \"$VERSION\",
+    \"gpgSign\": false
   },"
 
-case $1 in
+case "$BINTRAY_REPO_NAME_SUFFIX" in
   "debian")
+    # Note: this refers to the latest released Ubuntu, 16.04 LTS (Xenial Xerus).
+
+    # TODO: use relative includePatterns.
+
     FILES="\"files\":
         [
           {
-            \"includePattern\": \"build/bin/(.*\.deb)\", \"uploadPattern\": \"\$1\",
+            \"includePattern\": \"\\\\/home\\\\/travis\\\\/build\\\\/ponylang\\\\/ponyc\\\\/build\\\\/bin\\\\/(.*\\\\.deb)\", \"uploadPattern\": \"\$1\",
             \"matrixParams\": {
-            \"deb_distribution\": \"vivid\",
+            \"deb_distribution\": \"xenial\",
```

Comment:
I hope this field doesn't limit which Debian and Ubuntu versions will be able to use these DEB files created by FPM...